.github/workflows/release.yaml

```diff
@@ -1,13 +1,14 @@
 name: Release
 
 on:
+  workflow_dispatch:
   push:
-    tags: ["v*"]
```

Comment:
I assume we'll still want to run the release workflow on tag as well?

Reply:
Not sure -- tags are just annotations on existing commits. If we don't plan on tagging commits that exist only on specific branches, we should be fine. If we want to have tags, and if we want a special build for that, we can trigger it manually after pushing the tag. `git describe` should then return the proper version based on the tag.

Reply:
Created https://github.com/grafana/opentelemetry-collector-components/issues/50 to discuss